### ISSUE TITLE
Region update from serverless.yml

### ISSUE THREAD

**Issue by Dalfons0** [elastic-coders/serverless-webpack#89]
I'm not sure if this is an issue but I think is a good addition. The thing is that if you try to invoke a function that access an aws service (e.g. DynamoDB), with `serverless webpack invoke` or `serverless webpack serve` and you haven't specified the region in the handler like `AWS.config.region = "us-west-2"` or `AWS.config.update({region: "us-west-2"})`, you got this error:
```
  Config Error -------------------------------------------

     Missing region in config
```
And this doesn't happen if you invoke a function with the command `serverless invoke local` who I suppose  is getting the region from serverless.yml

**Comment by hassankhan** [comment]
This is happening for me too, I've defined my stage and region in `serverless.yml`.

**Comment by hassankhan** [comment]
I've posted my workaround for this issue [here](https://github.com/dherault/serverless-offline/issues/189).

**Comment by Dalfons0** [comment]
@hassankhan Thanks for the info!

**Comment by HyperBrain** [comment]
@Dalfons0 We'll switch the invoke command to use `serverless invoke local` with version 3.0.0. A working PR is already available ( #153 ). Could you check if it works with that source code revision if you use `serverless invoke local`?

**Comment by HyperBrain** [comment]
`invoke` will be removed in the next `v3.0.0` release and integrates fully with `serverless invoke local` (see #151 ). No efforts will be spent on the function anymore. So I'll close this issue.
